ncm-download/src/main/pan/components/download/schema.pan

```diff
@@ -43,6 +43,9 @@ type component_download_file = {
     "allow_older" ? boolean
 };
 
+@{safety check against shell metacharacters, for kinit_args}
```

Comment:
shell metacharcters? the kinit commands are not executed in a subshell, so you shouldn't worry about these

Reply:
@stdweird - ah, so you are saying it's safe to be just a string type as @natari originally had it?

Reply:
@ned21 i think so. but @natari originally had a string with some odd regexp i tought were releated to eg kinit commands or principals, not to avoid shell escaping

```perl
#!/usr/bin/perl

BEGIN {push(@INC, '/usr/lib/perl')}

use CAF::Process;
print CAF::Process->new(['echo', '$PATH'])->output();
print CAF::Process->new(['echo', 'abc;echo hellobash'])->output();
```
gives
```
$PATH
abc;echo hellobash
```
so no shell involved (unless this is not what you mean)

Reply:
@stdweird yeah my original regexp was to avoid shell metacharacters. It does look like CAF::Process doesn't need it, so I will remove the regexp and leave just a string[] for the args. Thanks!